community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/path/ShortestPath.java

```diff
@@ -162,7 +162,7 @@ private void resolveMonitor( Node node )
                 goOneStep( endData, startData, hits, startData, stopAsap );
             }
             Collection<Hit> least = hits.least();
-            return least != null ? filterPaths( hitsToPaths( least, start, end, stopAsap ) ) : Collections.emptyList();
+            return least != null ? filterPaths( hitsToPaths( least, start, end, stopAsap, maxResultCount ) ) : Collections.emptyList();
```

Comment:
would be nice to see test for this change

Reply:
Well, TestShortestPath tests this. It's now green.

Reply:
Since we're gonna run one more build w/ the println removal anyway, I can create an addition test perhaps